[2024-09-19T18:13:48.437+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-09-18T00:00:00+00:00 [queued]>
[2024-09-19T18:13:48.569+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-09-18T00:00:00+00:00 [queued]>
[2024-09-19T18:13:48.585+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-09-19T18:13:49.678+0000] {taskinstance.py:1380} INFO - Executing <Task(PinotSchemaSubmitOperator): submit_schemas> on 2024-09-18 00:00:00+00:00
[2024-09-19T18:13:51.250+0000] {standard_task_runner.py:57} INFO - Started process 433 to run task
[2024-09-19T18:13:51.354+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'schema_dag', 'submit_schemas', 'scheduled__2024-09-18T00:00:00+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/schema_dag.py', '--cfg-path', '/tmp/tmp0gu1dzrw']
[2024-09-19T18:13:51.710+0000] {standard_task_runner.py:85} INFO - Job 39: Subtask submit_schemas
[2024-09-19T18:14:01.788+0000] {task_command.py:415} INFO - Running <TaskInstance: schema_dag.submit_schemas scheduled__2024-09-18T00:00:00+00:00 [running]> on host 48913e9acee9
[2024-09-19T18:14:31.897+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='codewithyu' AIRFLOW_CTX_DAG_ID='schema_dag' AIRFLOW_CTX_TASK_ID='submit_schemas' AIRFLOW_CTX_EXECUTION_DATE='2024-09-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-18T00:00:00+00:00'
[2024-09-19T18:14:35.140+0000] {pinot_schema_operator.py:35} ERROR - An error occurred: HTTPConnectionPool(host='pinot-controller', port=9000): Max retries exceeded with url: /schemas (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffff85a7e220>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2024-09-19T18:14:35.673+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=schema_dag, task_id=submit_schemas, execution_date=20240918T000000, start_date=20240919T181348, end_date=20240919T181435
[2024-09-19T18:14:40.261+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-09-19T18:14:44.298+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
